src/main/resources/liquibase/crop_changelog/24_3_0.xml

```diff
@@ -0,0 +1,390 @@
+    </changeSet>				
+				
+    <changeSet id="v24.3.0-9" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "Assigned Passport Descriptor" AND cv_id = 1020;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add cvterm entry for Method Assigned Passport Descriptor</comment>
+        <sql dbms="mysql" splitStatements="true">
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1020, 'Assigned Passport Descriptor', 'Standard method for assigned passport descriptor', null, 0, 0);
+        </sql>
+    </changeSet>
+
+    <changeSet id="v24.3.0-10" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "Text" AND cv_id = 1030;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add cvterm entry for Scale Text</comment>
+        <sql dbms="mysql" splitStatements="true">
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1030, 'Text', 'Text', null, 0, 0);
+        </sql>
+    </changeSet>   
+	
+	<changeSet id="v24.3.0-11" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "PLOT_NUMBER_AP_text" and cv_id = 1040 ;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add variable for Packing _Status</comment>
+        <sql dbms="mysql" splitStatements="true">
+            set @methodID = (select cvterm_id from cvterm where name = 'Assigned Passport Descriptor' AND cv_id = 1020);
+            set @scaleId = (select cvterm_id from cvterm where name = 'Text' AND cv_id = 1030);
+			set @propertyId = (select cvterm_id from cvterm where name = 'Plot Number' AND cv_id = 1010);
+			set @class = (SELECT cvterm_id FROM cvterm WHERE name = "Passport" AND cv_id = 1011);
+			
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1040, 'PLOT_NUMBER_AP_text', 'Plot Number', null, 0, 0);
+
+            set @variableId = (select cvterm_id from cvterm where name = 'PLOT_NUMBER_AP_text' AND cv_id = 1040);
+			
+			INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1225, @propertyId, @class );
+			
+            INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1200, @variableId, @propertyId);
+
+            INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1210, @variableId, @methodId);
+
+            INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1220, @variableId, @scaleId);
+
+            INSERT INTO cvtermprop(cvterm_id, type_id, value, rank)
```

Comment:
@rvtulod we also need to insert a row for the creation date in cvtermprop with type_id = 8048